### DISCUSSION TITLE
Add a setting that allows users to choose if scroll wheel changes the volume by 1, 2 or 5

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#17418] [Ideas] [answered]
In my opinion this would make the change from Stable more fluid, as in Stable the scroll wheel changes the volume by 5, whereas in Lazer it changes the volume by 1

**Comment by peppy** [comment, marked as answer]
It will adjust by more if you scroll more. It has slight acceleration. Please give it a try.

**Comment by Skubers** [comment]
All right, I guess that makes it up for that

Pobierz aplikację Outlook dla systemu Android<https://aka.ms/AAb9ysg>
________________________________
